indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -171,10 +171,12 @@
 
 (defnestedmapping platform-hierarchical-mapping
   "Defines hierarchical mappings for platforms."
-  {:category m/string-field-mapping
+  {:basis m/string-field-mapping
+   :basis-lowercase m/string-field-mapping
+   :category m/string-field-mapping
    :category-lowercase m/string-field-mapping
-   :series-entity m/string-field-mapping
-   :series-entity-lowercase m/string-field-mapping
+   :sub-category m/string-field-mapping ;; formally :series-entity
```

Comment:
formally or formerly?
Also, can we alter existing mappings? I thought it was additive only and we'll need to re-index all collections and granules again.

Reply:
formerly. Just collections need to be remapped, which I was told happens every night?

Reply:
Right. We can't remove an existing field. We can still call it `series-entity` in ES and use it to store `sub-category` for now. This will require additional code change.